frontend/src/i18n/locales/en-US.json

```diff
@@ -43,7 +43,9 @@
     "off": "off",
     "about": "About",
     "built-by": "Built by {0}",
-    "links": "Links"
+    "links": "Links",
+    "send-history": "Send history",
+    "send-history-description": "Enables saving send history locally in the browser and being viewed on the sent page."
```

Comment:
```suggestion
    "send-history-description": 'Enables saving send history locally in the browser so it can be viewed on the "send history" page.'
```

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/560/commits/39556fb5222b2ab65d30292a47c4eea7159ff98e

Reply:
I will update the chinese translations once Gary comments